bigbluebutton-html5/imports/ui/components/nav-bar/container.jsx

```diff
@@ -28,6 +29,14 @@ export default withTracker(() => {
   const meetingObject = Meetings.findOne({
     meetingId,
   }, { fields: { 'meetingProp.name': 1, 'breakoutProps.sequence': 1 } });
+  const {findBreakouts}=BreakoutService;
+  const breakouts=findBreakouts();
+  let breakname;
+  breakouts.map((breakout) =>{breakout.users.map((user)=>{if(user.userId==Auth.userID){
```

Comment:
This is a much better formatting  - you should follow these standards

  let breakname = breakouts.map(breakout => {
    breakout.users.map(user => {
      if(user.userId==Auth.userID){
        breakname= breakout.name;
      }
    })
  });